.github/workflows/pr-filecheck.yml

```
@@ -45,18 +45,13 @@ jobs:
         with:
           node-version: "12"
 
-      - name: Get yarn cache directory path
-        if: steps.git_diff_content.outputs.diff
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v2.1.3
-        if: steps.git_diff_content.outputs.diff
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.3
+        id: cached-node_modules
         with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
+          path: |
+            node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
 
       - name: Install all yarn packages
         if: steps.git_diff_content.outputs.diff
```

This now doesn't match how `pr-build.yml` works. So it will execute `yarn --frozen-lockfile` every time this workflow runs. 

Is this even possible...
```suggestion
        if: steps.git_diff_content.outputs.diff && steps.cached-node_modules.outputs.cache-hit != 'true'
```